### PR TITLE
First pull request - Project "Hello Microverse!".

### PR DESCRIPTION
**Pull request by jcmunav63**
### General requirements

Once you finish your project, double-check to make sure your repo is ready:

- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- Make sure that you used correct ([GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md)).
- Make sure that you documented your work.

### Requirements

- Remember to document your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
- Create a new GitHub repo.
- Create a new branch according to the [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) rules.
- Add a `.gitignore` file.
- Add `node_modules` to gitignore file.
- Set up [a linter for HTML and CSS](https://github.com/microverseinc/linters-config) and make sure that any files that are not relevant are ignored by git.
- Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.
- Add 3 files (each one in a separate commit):
    - A .html file that includes a header with the text "Hello Microverse!". Use `h1` tag for that.
    - A .css file that includes one class with styles for your header (make it in your favorite color).
    - A test.md file with the text "This file should be ignored by git".
- Make sure that the last file is actually ignored by git and is not present in your GitHub repository.
- Open a pull request.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,233 @@
...
+<!---->
+
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse Hello README File</b></h3>
+
+</div>
+
```
Comment (TadesseDev) [inline README.md:12]: - [ ] Please consider Updating the Microverse logo with yours if possible or remove it totally. The template is designed by Microverse for us all to use to professionally document our projects and has it filled with content related to each project we may be working on.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,233 @@
...
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [microverse-hello] <a name="about-project"></a>
+
+> This is my first project called "Microverse Hello".
+> It relates with the subjects and activities of Day 2.
```
Comment (TadesseDev) [inline README.md:41]: - [ ] Please notice the section in your readme which start with `>` are directions/guidelines from microverse to help you understand what to write in that section and should not be part of your readme file. Please consider removing this and all the like contents from the readme file.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @jcmunav63 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7


## Highlights
 - [ ]  Correct GitHub Flow  ✅
 - [ ]  Good commit history  ✅
 
## Required Changes ♻️
 - [ ] You used the recommended README template for your project but at the moment you did not fill it with contents related to this project or your information. The template is designed by Microverse for us all to use to professionally document our projects and have it filled with content related to each project we may be working on. Kindly update your README to have real content for this project and your social handlers 👍
- [ ] Please make sure to set up linters correctly for this repository. [here you will find](https://github.com/microverseinc/linters-config/tree/master/html-css) a documentation on how to do that. we need to see a green `✅` check on your PR.
- [ ] Please try to be more descriptive in your pull request sumery. This is a place where you can introduce what changes you have made since the last merge. You can also specify any information that you would like to communicate with the code reviewer of the project.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (jcmunav63) [reply README.md:12]: Good day, Tadesse: I made all changes but I got confused in git with the branches and commands PUSH and MERGE. I think I accidentally merged from the main branch, which caused the Pull Request to close.
Can you explain to me what to do and how should I proceed (make corrections in the feature branch, then commit in that branch and then pull changes to the remote repo from the feature branch? Do I need to merge the main branch in my local repo?). Thank you.

Comment (TadesseDev) [reply README.md:12]: Dear @jcmunav63. I believe you can restore and reopen the branch. then you can reset your main branch to the earlier version of it `git reset --hard 'your last commit hash before the merge'` then force push your changes.

**Pull request merged**